### PR TITLE
needed changes

### PR DESCRIPTION
**Pull request by mtkat4k**

### CODE REVIEW

File: lib/sqspoller/worker_task.rb [hunk 1]
```diff
@@ -4,9 +4,6 @@
 module Sqspoller
   class WorkerTask
 
-    HEADERS = { 'Content-Type' => 'application/json',
```
Comment (anshul-fk) [inline lib/sqspoller/worker_task.rb]: Why have you removed this ?

File: lib/sqspoller/sqs_poller.rb [hunk 2]
```diff
@@ -77,6 +83,11 @@ def start_poller(filename, queue_config_name, access_key_id, secret_access_key,
         start_poller_from_content content, queue_config_name, access_key_id, secret_access_key, region, log_filename
       end
 
+      def start_poller_from_redis_config(redis_key, queue_config_name, access_key_id, secret_access_key, region, log_filename=nil)
+        content = REDIS.get ENV['REDIS_CONFIG_KEY']
```
Comment (anshul-fk) [inline lib/sqspoller/sqs_poller.rb]: JSON.parse

**Review by anshul-fk — CHANGES_REQUESTED** [review]

File: lib/sqspoller/worker_task.rb (hunk 1 above)
Comment (mtkat4k) [reply lib/sqspoller/worker_task.rb]: it was moved upstream in a change that has not been committed yet...

File: lib/sqspoller/sqs_poller.rb (hunk 2 above)
Comment (mtkat4k) [reply lib/sqspoller/sqs_poller.rb]: this feeds into the same handler as starting from file which does the ERB and Yaml parsing

**Pull request closed without merging**